initializer_meta.go

```diff
@@ -1,13 +1,18 @@
 package process
 
-import "time"
+import (
+	"time"
+
+	"github.com/go-nacelle/log"
+)
 
 type (
 	// InitializerMeta wraps an initializer with some package
 	// private fields.
 	InitializerMeta struct {
 		Initializer
 		name            string
+		loggingFields   log.LogFields
```

Comment:
Can this be `logFields` instead of `loggingFields` (same for the other places as well)? It seems like an unnecessarily long name given logging is referred to as "log" everywhere (or almost everywhere) and `logFields` would match with the name of the type being passed in as well.

Reply:
Yes, but I'll have to rename things inside of the nacelle meta package as well (which is fine).